<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitulo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Sistema de Ordenação Usando FIla&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExecutar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>260</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditVetorOrdenado">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>581</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelVetorOrdenado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>181</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Vetor Ordenado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Crescente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decrescente</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelRegra">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#000000;&quot;&gt;Regra&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVetorOriginal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>161</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#000000;&quot;&gt;Vetor Original&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditVetorOriginal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>559</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuantidadeElemento">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>227</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#000000;&quot;&gt;Quantidade de Elementos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuantidadeElementos">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
